bin/vitamin.js

```diff
@@ -190,7 +188,7 @@ const start = (options) => {
         exiting = true;
         if (!serverProcess) process.exit();
         killProcess(serverProcess, { signal }).then(() => {
-            console.log('exiting'); process.exit();
+            logger.log('Exiting...'); process.exit();
```

Comment:
Why do you leave a `...` if you exit right away ? `...` could mean that something else should have been printed after